packages/dht/src/connection/WebRTC/WebRtcConnector.ts

```diff
@@ -149,11 +148,11 @@ export class WebRtcConnector implements IWebRtcConnectorService {
         })
 
         if (offering) {
-            connection.once('localDescription', (description: string, _type: string) => {
+            connection.once('localDescription', (description: string) => {
```

Comment:
Might be useful to check the type field here to ensure that the type of the description is `offer` or `answer` when appropriate